plugins/Transitions/API.php

```diff
@@ -55,20 +55,23 @@ public function getTransitionsForPageUrl($pageUrl, $idSite, $period, $date, $seg
      * @param $period
      * @param $date
      * @param bool $segment
-     * @param bool $limitBeforeGrouping
+     * @param int $limitBeforeGrouping
      * @param string $parts
      * @return array
      * @throws Exception
      */
     public function getTransitionsForAction($actionName, $actionType, $idSite, $period, $date,
-                                            $segment = false, $limitBeforeGrouping = false, $parts = 'all')
+                                            $segment = false, $limitBeforeGrouping = 0, $parts = 'all')
```

Comment:
I guess it would make sense to also change that for all other API methods, otherwise the same error could occur there as well.
And it might also make sense to check and change all methods were that parameter is passed through, as some of them also use `false` as default...

Reply:
@sgiehl that makes sense, update a couple of places, and I notice when $limitBeforeGrouping is given as a string `'1'` or `'20'` is will state as an error as well, added `intval `as well.